kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/test/timeout/CoroutinesTimeoutTest.kt

```diff
@@ -88,3 +95,20 @@ private suspend fun someCoroutine() {
       }
    }
 }
+
+/** Delay using real (non-virtual) time. */
+suspend fun wallclockDelay(duration: Duration) {
```

Comment:
should we rename this function to something like `nonVirtualTimeDelay` to design away the need for this comment?